setup.py

```diff
@@ -27,8 +27,8 @@
     packages=find_namespace_packages(where="src", exclude=("test",)),
     package_dir={"": "src"},
     install_requires=[
-        "amazon-braket-schemas>=1.10.1",
-        "amazon-braket-default-simulator>=1.7.2",
+        "amazon-braket-schemas",
+        "amazon-braket-default-simulator",
```

Comment:
I understand that it's convenient for you to have this in your local copy, but there is no reason to commit it. Please reverse it in the PR.